public/index.html

```diff
@@ -24,7 +24,18 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+
+    <!-- Rapid UI -->
+    <link rel="stylesheet" href="https://rapidui.netlify.app/styles/main.css" />
+    <link
```

Comment:
Font Awesome is fine but you can also try react-icons they are totally free and has a lot of variety.